scripts/fuzz_opt.py

```diff
@@ -603,7 +603,7 @@ def test_one(random_input, opts, given_wasm):
     print('pre wasm size:', wasm_size)
 
     # create a second wasm for handlers that want to look at pairs.
-    generate_command = [in_bin('wasm-opt'), 'a.wasm', '-o', 'b.wasm'] + opts + FUZZ_OPTS + FEATURE_OPTS
+    generate_command = [in_bin('wasm-opt'), 'a.wasm', '-o', 'b.wasm'] + FUZZ_OPTS + opts + FEATURE_OPTS
```

Comment:
`FUZZ_OPTS` might contain a pass to do JS legalization, which can only run on normal IR rather than stack IR. Running `FUZZ_OPTS` after `opts` crashes when that is the case and `opts` ends with Stack IR. Changing their order was a simple workaround.